x-pack/plugins/rule_registry/server/rule_data_plugin_service/index.ts

```diff
@@ -153,6 +156,13 @@ export class RuleDataPluginService {
   }
 
   getFullAssetName(assetName?: string) {
-    return [this.options.index, assetName].filter(Boolean).join('-');
+    return [this.fullAssetName, assetName].filter(Boolean).join('-');
+  }
+
+  async assertFullAssetNameExists(assetName?: string) {
```

Comment:
This appears to be unused?

Reply:
Yup not used anymore. Fixed here: 6111991397dd51f51afab7c6a2cda30db267b357